libsolidity/ast/ASTJsonConverter.cpp

```diff
@@ -704,13 +705,13 @@ bool ASTJsonConverter::visit(ElementaryTypeNameExpression const& _node)
 
 bool ASTJsonConverter::visit(Literal const& _node)
 {
-	char const* tokenString = Token::toString(_node.token());
+	string tokenString = tokenKind(_node.token());
 	Json::Value value{_node.value()};
 	if (!dev::validateUTF8(_node.value()))
 		value = Json::nullValue;
 	Token::Value subdenomination = Token::Value(_node.subDenomination());
 	std::vector<pair<string, Json::Value>> attributes = {
-		make_pair("token", tokenString ? tokenString : Json::Value()),
+		make_pair("kind", tokenString),
```

Comment:
This change seems to be a bugfix to the current printing. Perhaps we could pull it into a separate PR and merge it now?